scripts/verify-replace.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+TMP_DIR=/tmp/
+BASE_REPO_PATH=$(mktemp -d ${TMP_DIR}replace-verify.XXX)
+GH_BASE_URL_KS=https://github.com/kubesaw/
+GH_BASE_URL_CRT=https://github.com/codeready-toolchain/
+declare -a repos=("${GH_BASE_URL_KS}ksctl" "${GH_BASE_URL_CRT}host-operator" "${GH_BASE_URL_CRT}member-operator" "${GH_BASE_URL_CRT}registration-service" "${GH_BASE_URL_CRT}toolchain-e2e")
```

Comment:
it's just a detail, but global variables and constants are usually ALL_CAPS,  while local can be lower_caps
```suggestion
declare -a REPOS=("${GH_BASE_URL_KS}ksctl" "${GH_BASE_URL_CRT}host-operator" "${GH_BASE_URL_CRT}member-operator" "${GH_BASE_URL_CRT}registration-service" "${GH_BASE_URL_CRT}toolchain-e2e")
```

Reply:
updated it